lib/block-supports/background.php

```diff
@@ -103,4 +103,7 @@ function gutenberg_render_background_support( $block_content, $block ) {
 	)
 );
 
+if ( function_exists( 'wp_render_background_support' ) ) {
+	remove_filter( 'render_block', 'wp_render_background_support' );
```

Comment:
Should we also include the priority args?

https://github.com/WordPress/wordpress-develop/blob/5eef2bbb2a476fc3f94f7d7c2d89ff01b9b74f82/src/wp-includes/block-supports/background.php#L118

Actually, it might be optional: https://developer.wordpress.org/reference/functions/remove_filter/

Reply:
Yeah, we don't need the priority args for this one since it's registered with the default `10`.